### PR TITLE
Fixes the api since changes from last week introduced a new bug.

### PR DESCRIPTION
**Pull request by sebastian-nehrdich**

### CODE REVIEW

File: api/main.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ async def get_parallels_for_middle(parallels: ParallelsCollection):
     query_result = get_db().AQLQuery(
         query=main_queries.QUERY_PARALLELS_FOR_MIDDLE_TEXT,
         batchSize=10000,
-        bind_vars={
+        bindVars={
```
Comment (ayya-vimala) [inline api/main.py:93]: Why did you change all these? CamelCase is for Javascript.

File: api/main.py [hunk 2]
```diff
@@ -260,14 +260,15 @@ async def get_files_for_menu(language: str):
     """
     menu_query = menu_queries.QUERY_FILES_FOR_LANGUAGE
     current_bind_vars = {"language": language}
+    print("CURRENT_BIND_VARS",current_bind_vars)
```
Comment (ayya-vimala) [inline api/main.py:263]: Print statement

File: api/main.py [hunk 3]
```diff
@@ -633,9 +634,10 @@ async def get_data_for_sidebar_menu(language: str):
     """
     Endpoint for sidebar menu
     """
+    print("SIDEBAR LANG",language)
```
Comment (ayya-vimala) [inline api/main.py:637]: Print statement

File: api/main.py (hunk 2 above)
Comment (ayya-vimala) [reply api/main.py:263]: This is also the Travis error.

**Review by ayya-vimala — CHANGES_REQUESTED** [review]
If you want to leave the bindVars that is fine but please remove the print statements. That will also take care of the Travis error.

File: api/main.py (hunk 1 above)
Comment (sebastian-nehrdich) [reply api/main.py:93]: this unfortunately is due to the nature of python arangodb bindings. They require the camelCasing, we can't really change it :(

**Review by ayya-vimala — APPROVED** [review]
